src/index.html

```diff
@@ -13,6 +13,38 @@
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <ul class="calendar calendar--start-day-sun calendar--month-length-31">
```

Comment:
in this way we use `modifiers`
```suggestion
    <ul class="calendar calendar--start-day--sun calendar--month-length--31">
```